controllers/account/account_controller_test.go

```diff
@@ -1549,7 +1549,10 @@ var _ = Describe("Account Controller", func() {
 							},
 						},
 					}, nil)
-					err := setCurrentAccountServiceQuotas(r, nullLogger, account, mockAWSClient)
+					regionsEnabledInAccount, _ := mockAWSClient.DescribeRegions(&ec2.DescribeRegionsInput{
+						AllRegions: aws.Bool(false),
+					})
+					err := SetCurrentAccountServiceQuotas(nullLogger, account, regionsEnabledInAccount)
```

Comment:
Wasn't sure want to change this to. The original line :  `err := setCurrentAccountServiceQuotas(r, nullLogger, account, mockAWSClient)` .  The function was updated

Reply:
The mock that is being used there is just returning the following `DescribeRegionsOutput`:

```
&ec2.DescribeRegionsOutput{
						Regions: []*ec2.Region{
							{
								RegionName: aws.String("us-east-1"),
							},
						},
					}, nil)
```

Given that this is now passed directly, you should be able to remove the mockClient completely from this test and just pass in the DescribeRegionsOutput directly.